tests/fixups/TraceFixup/main.cpp

```diff
@@ -151,119 +156,149 @@ result_configuration configured_result(function_type type, function_result resul
 }
 
-void Log_ETW_PostMsgOperationA(const char *operation, const char *inputs, const char *result, const char *outputs, const char *callingmodule, LARGE_INTEGER TickStart, LARGE_INTEGER TickEnd)
+void Log_ETW_PostMsgOperationA(const char* operation, const char* inputs, const char* result, const char* outputs, const char* callingmodule, LARGE_INTEGER TickStart, LARGE_INTEGER TickEnd)
 {
-	TraceLoggingWrite(g_Log_ETW_ComponentProvider, // handle to my provider
-		"TraceEvent",              // Event Name that should uniquely identify your event.
-		TraceLoggingValue(operation, "Operation"),
-		TraceLoggingValue(inputs, "Inputs"),
-		TraceLoggingValue(result, "Result"),
-		TraceLoggingValue(outputs, "Outputs"),
-		TraceLoggingValue(callingmodule, "Caller"), 
-		TraceLoggingInt64(TickStart.QuadPart, "Start"),
-		TraceLoggingInt64(TickEnd.QuadPart, "End")
-	); // Field for your event in the form of (value, field name).
+    TraceLoggingWrite(g_Log_ETW_ComponentProvider, // handle to my provider
+        "TraceEvent",              // Event Name that should uniquely identify your event.
+        TraceLoggingValue(operation, "Operation"),
+        TraceLoggingValue(inputs, "Inputs"),
+        TraceLoggingValue(result, "Result"),
+        TraceLoggingValue(outputs, "Outputs"),
+        TraceLoggingValue(callingmodule, "Caller"),
+        TraceLoggingInt64(TickStart.QuadPart, "Start"),
+        TraceLoggingInt64(TickEnd.QuadPart, "End")
+    ); // Field for your event in the form of (value, field name).
 }
 
-void Log_ETW_PostMsgW(const wchar_t * s)
+void Log_ETW_PostMsgW(const wchar_t* s)
 {
-	TraceLoggingWrite(g_Log_ETW_ComponentProvider, // handle to my provider
-		"TraceEvent",              // Event Name that should uniquely identify your event.
-		TraceLoggingValue(s, "Message")); // Field for your event in the form of (value, field name).
+    TraceLoggingWrite(g_Log_ETW_ComponentProvider, // handle to my provider
+        "TraceEvent",              // Event Name that should uniquely identify your event.
+        TraceLoggingValue(s, "Message")); // Field for your event in the form of (value, field name).
 
 }
 
 // Tear down the ETW Provider
 void Log_ETW_UnRegister()
 {
-	TraceLoggingUnregister(g_Log_ETW_ComponentProvider);
+    TraceLoggingUnregister(g_Log_ETW_ComponentProvider);
 }
 
 BOOL __stdcall DllMain(HINSTANCE, DWORD reason, LPVOID) noexcept try
 {
     if (reason == DLL_PROCESS_ATTACH)
     {
+        Log_ETW_Register();
+
+        std::wstringstream traceDataStream;
+
         if (auto config = PSFQueryCurrentDllConfig())
         {
             auto& configObj = config->as_object();
 
+            traceDataStream << "config : \n";
             if (auto method = configObj.try_get("traceMethod"))
             {
                 auto methodStr = method->as_string().string();
+                traceDataStream << " traceMethod : " << method->as_string().wide() << " ;";

```

Comment:
>  [](start = 87, length = 1)

Are the spaces before the semicolon intentional? This will cause format to be "foo : bar ;"